src/Exceptionless.Web/Controllers/AdminController.cs

```diff
@@ -112,7 +169,7 @@ public class AdminController : ExceptionlessApiController {
         public async Task<IActionResult> RunJobAsync(string name) {
             switch (name.ToLowerInvariant()) {
                 case "indexes":
-                    if (!Settings.Current.DisableIndexConfiguration)
+                    if (!AppOptions.Current.DisableIndexConfiguration)
```

Comment:
Why are we using AppOptions here when we have access to _appOptions?

Reply:
overlooked